packages/@o3r/eslint-config-otter/migration.json

```diff
@@ -5,6 +5,11 @@
       "version": "10.0.0-alpha.0",
       "description": "Updates of @o3r/eslint-config-otter to v10.0.*",
       "factory": "./schematics/ng-update/index#updateV100"
+    },
+    "migration-v11_4": {
```

Comment:
maybe you want to target the `release/11.4.0-rc` branch instead of `main`

Reply:
nop `main` is the right target but the PR was created before the `rc`.
I changed for `11.5` instead